lesson15/main.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+const wrapper = document.getElementById("js-wrapper");
+const ul = document.getElementById("js-ul");
+const submitBtn = document.getElementById("js-submitBtn");
+submitBtn.classList.add("btn");
+const modal = document.getElementById("js-modal");
+const openModalBtn = document.getElementById("js-openModalBtn");
+const jsonURL = "https://myjson.dit.upm.es/api/bins/j16z";
+const removeModal = () => {
+   modal.remove();
+   wrapper.classList.remove("bg_gray");
+};
+
+const startLoading = () => {
+   const loadingIcon = document.createElement("img");
+   loadingIcon.id = "loadingIcon";
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+};
+
+const endLoading = () => {
+   const loadingIcon = document.getElementById("loadingIcon");
+   loadingIcon.remove();
+}
+
+openModalBtn.addEventListener("click", function () {
+   modal.style.display = "block";
+   wrapper.classList.add("bg_gray");
+   openModalBtn.remove();
+});
+
+const isEmptyString = (value) => value === "";
+
+submitBtn.addEventListener("click", () => {
+   const inputNumber = document.getElementById("js-number");
+   const inputText = document.getElementById("js-text");
+   if (isEmptyString(inputNumber.value) || isEmptyString(inputText.value)) { //inputNumber.valueとinputText.valueのデータ型がStringのため
+      alert("未入力です。");
+      return;
+   }
+   console.log('ID:' + inputNumber.value);
+   console.log('ユーザー名:' + inputText.value);
```

Comment:
nits
[Template literals](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals) can be used to write.
```suggestion
  console.log(`ID:${inputNumber.value}`);
  console.log(`ユーザー名:${inputText.value}`);
```

Reply:
Thank you for your comment.
I changed the writing style to use template literals.
11a73af